Packs/Phishing/IncidentFields/incidentfield-Attachment_ID.json

```diff
@@ -24,7 +24,8 @@
  "group": 0,
  "hidden": false,
  "associatedTypes": [
-  "Phishing"
+  "Phishing",
+  "Email Communication"
  ],
  "systemAssociatedTypes": [
```

Comment:
In order for the association to work, please add to "systemAssociatedTypes" as well for all other fields as well

Reply:
The email communication incident type is already in the "systemAssociatedTypes" for the relevant incident fields